### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by duncangrant**

### CODE REVIEW

File: core/src/main/java/brooklyn/location/cloud/CloudLocationConfig.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
 
     public static final ConfigKey<String> WAIT_FOR_SSHABLE = ConfigKeys.newStringConfigKey("waitForSshable", 
             "Whether and how long to wait for a newly provisioned VM to be accessible via ssh; " +
-            "if 'false', won't check; if 'true' uses default duration; otherwise accepts a time string e.g. '5m' (the default) or a number of milliseconds", "5m");
+            "if 'false', won't check; if 'true' uses default duration; otherwise accepts a time string e.g. '5m' (the default) or a number of milliseonds", "5m");
```
Comment (nakomis) [inline core/src/main/java/brooklyn/location/cloud/CloudLocationConfig.java]: Introduces typo

File: README.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ Alternatively, use Brooklyn as an integrated-stand-alone management node for you
 
 Three quick start options are available:
 
-* The [getting started guide](http://brooklyncentral.github.io/use/guide/quickstart/index.html) will step you through downloading and installing Brooklyn and running the examples.
+* The [getting started guide](https://brooklyn.incubator.apache.org/quickstart/) will step you through downloading and installing Brooklyn and running the examples.
 * Alternatively, [download the latest release](https://github.com/brooklyncentral/brooklyn/tarball/master) (tgz),
-* or, fork or clone the repo: `git clone git://github.com/brooklyncentral/brooklyn.git` then `mvn clean install`.
+* or, fork or clone the repo: `git clone git://github.com/apache/incubator-brooklyn.git` then `mvn clean install`.
```
Comment (nakomis) [inline README.md]: `cd incubator-brooklyn` required, or even `cd incubator-brooklyn; mvn clean install`

**Comment by nakomis** [comment]
URLs tested, LGTM

**Comment by sjcorbett** [comment]
Test failures irrelevant. Merging.

**Pull request merged**